src/stories/input/appearence.stories.tsx

```diff
@@ -51,7 +67,7 @@ WithClearButton.args = {
   value: "text",
   hasClearButton: true,
   placeholder: "Type something",
-  onChange: (e) => {
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
would it make sense to move this type to the component and rely on type inference here instead?